README.md

```diff
@@ -477,6 +477,12 @@ All statements above will return false if used with `===`
 
 🔗 [**Read More: Refactoring!**](/sections/testingandquality/refactoring.md)
 
+## ![✔] 4.10 Run tests in parallel
+
+**TL;DR:** Design your tests so you can run them concurrently. Your total test times will be dramatically lower since the runner can take advantage of multiple CPU cores and slow tests won't block execution. Consider a framework that supports parallelization out of the box. Some run multiple tests in the same file in parallel ([Ava](https://github.com/avajs/ava)), whereas others only run multiple files in parallel ([Jest](https://jestjs.io), [mocha-parallel-tests](https://github.com/mocha-parallel/mocha-parallel-tests)). The latter is simpler for development, but more limiting. Even if your tests are currently small, it is difficult to parallelize existing tests, so if your project might get large, starting with parallel execution helps.
```

Comment:
Why is it difficult to parallelize existing tests? this part deserves some clarification

#just-an-idea